src/scripts/index.js

```diff
@@ -0,0 +1,6 @@
+import StackedBarTrack from './StackedBarTrack';
+import ScaledStackedBarTrack from './ScaledStackedBarTrack';
+import BasicMultipleLineChart from './BasicMultipleLineChart';
+import BasicMultipleBarChart from './BasicMultipleBarChart';
+
```

Comment:
This index file serves no real purpose aside from testing & possibly allowing `import { ScaledStackedBarTrack, StackedBarTrack } from 'higlass-multivec';` re: `"module" : "./es"` in package.json (haven't tested this from 4dn-side yet actually.. (vs `..from 'higlass-multivec/es/<filename>';`))